website/docs/patterns/disabled-patterns/partials/accessibility.md

```diff
@@ -0,0 +1,21 @@
+The `disabled` attribute is only valid for these HTML elements, referred to as form controls:
+
+- `<button>`
+- `<fieldset>`
+- `<optgroup>`
+- `<option>`
+- `<select>`
+- `<textarea>`
+- `<input>`
+
+In terms of accessibility, however, a form control with the `disabled` attribute is problematic. If a form control has the `disabled` attribute, it's not available to users with assistive technology. It's as though the element does not exist in the DOM at all.
+
+When presenting a non-editable field, consider using the `readonly` attribute instead of the `disabled` attribute. This will provide the correct information to all users, but they won't be able to interact with it.
+
+If it's text that will eventually be a link (think, download link), then it should be text until it is valid to become a link (that's why the `disabled` attribute is not valid for `<a>` elements!). This can be accomplished with use of [conditionals](https://guides.emberjs.com/release/components/conditional-content/) in Ember.
+
+## Applicable WCAG Success Criteria
+
+The following Web Content Accessibility Guidelines (WCAG) may be relevant when considering the use of the `disabled` attribute:
```

Comment:
"may be" sounds wishy washy...

How about something like:
"Consult the following Web Content Accessibility Guidelines (WCAG) when considering using the `disabled` attribute:"

Reply:
I like this, will be editing because Melanie is OOO.